src/models/m_identity.erl

```diff
@@ -542,22 +543,34 @@ get_rsc(Id, Type, Context) ->
     z_db:assoc_row("select * from identity where rsc_id = $1 and type = $2", [Id, Type], Context).
 
 
-%% @doc Hash a password, using sha1 and a salt
+%% @doc Hash a password, using bcrypt
 %% @spec hash(Password) -> tuple()
 hash(Pw) ->
-    Salt = binary_to_list(z_ids:id(10)),
-    Hash = crypto:hash(sha, [Salt,Pw]),
-    {hash, Salt, Hash}.
+    {bcrypt, erlpass:hash(Pw)}.
 
 
+%% @doc Hash a password, with bcrypt
+%% @spec bcrypt_hash(Password) -> tuple
+
 %% @doc Compare if a password is the same as a hash.
 %% @spec hash_is_equal(Password, Hash) -> bool()
+hash_is_equal(Pw, {bcrypt, Hash}) ->
```

Comment:
As the tests still pass, I *think* you can ignore these errors for now. We also have them [on Travis](https://travis-ci.org/zotonic/zotonic/jobs/154162331): they seem to be caused by a boot order issue. Cf. #1176.